params/version.go

```diff
@@ -6,3 +6,6 @@ var Version string
 
 // GitCommit is a commit hash.
 var GitCommit string
+
+// IPFS_GATEWAY_URL is the Gateway URL to use for IPFS
+var IPFS_GATEWAY_URL string
```

Comment:
Why is this param names using the bash env variable style while other params here are camel-case?

Reply:
I will take the liberty of fixing this before merging.

Reply:
Even the linter is on my side:
```
params/version.go:11:5: don't use ALL_CAPS in Go names; use CamelCase (golint)
var IPFS_GATEWAY_URL string
    ^
```